arch/x86/core/x86_mmu.c

```diff
@@ -1906,6 +1906,7 @@ void arch_reserved_pages_update(void)
 		case X86_MEMMAP_ENTRY_DEFECTIVE:
 			__fallthrough;
 		default:
+			/* If none of the other cases are satisfied, leave the switch */
```

Comment:
Since 3 of the cases use `__fallthrough` to drop down into the default and also leave the switch, I don't know if the comment "if none of the other cases are satisfied, leave the switch" is accurate. Maybe "If any of three above cases satisfied, exit switch and mark page reserved" to give more information. That might be too much information / I could be nitpicking, though, so I'd like to see what others think :thinking:

Reply:
I will follow your advice.